### ISSUE TITLE
Add visibility selection (to control ACL) to the uploader

### ISSUE THREAD

**Issue by LukasKalbertodt** [elan-ev/tobira#369]
Similarly to #356, but before ingesting a video. Blocked by:

- #671 

Open question:
- What to do when user uploads into a series?

**Comment by LukasKalbertodt** [comment]
This is done in https://github.com/elan-ev/tobira/pull/911 / https://github.com/elan-ev/tobira/pull/947

Will close this. @dagraf could you create a separate issue with your exact requirements?
